test/state/state.cpp

```diff
@@ -131,6 +131,10 @@ void finalize(
                 return acc.erasable && acc.is_empty();
             });
     }
+
+    // Apply withdrawals. Amount value is in gwei.
+    for (const auto& withdraw : withdrawals)
+        state.touch(withdraw.first).balance += intx::uint256{withdraw.second} * 1000000000;
```

Comment:
Not sure if this is better but avoids counting zeroes:
```suggestion
        state.touch(withdraw.first).balance += intx::uint256{withdraw.second} * 1e9;
```

Not sure what is our preference in the codebase, @chfast.